frontend/src/lib/components/ChildrenRegistration.svelte

```diff
@@ -349,18 +351,18 @@ async function submitData(): Promise<void> {
 							bind:value={answers[element.id].answer}
 							bind:additionalValue={answers[element.id]
 								.additional_answer}
-							label={element?.text[$locale].question}
+							label={element?.text?.[$locale]?.question ?? ''}
```

Comment:
it is indeed, I removed a couple of them again, ignoring the raised issues because it doesn´t make much sense that `$locale` would be `null` or `undefined`. but the top level wrapper is a nice idea.